### ISSUE TITLE
getSettings() returns table data, not settings object

### ISSUE THREAD

**Issue by ottosson** [handsontable/angular-handsontable#219]
### Description
When I try to get the current settings object I use hot.getSettings(), but now it returns something that look a whole lot more like the table data array. This does not seem to be an issue in vanilla HoT.

### Steps to reproduce

1. use hot.getSettings();

### Demo

https://stackblitz.com/edit/handsontable-angular-fkshai?file=app%2Fapp.component.ts

### Your environment
* Angular version: 7.2
* Handsontable version: 8.1.0
* Wrapper version: 6.0.1
* Browser Name and version: latest Firefox, Chrome
* Operating System: Windows 10

PS. The dependencies in your default Stackblitz template are quite old and it would be nice if they were up to date. Thanks!

**Comment by AMBudnik** [comment]
Hi @ottosson 

I got some errors in the console so I've created a new demo based on Angular and added the `getSettings()` call inside a button. It seems to print exactly what we get from the vanilla version https://jsfiddle.net/AMBudnik/ugLyv9tr/ 
And here https://jsfiddle.net/AMBudnik/b8o7da96/ is the call to `getSettings()` via `ngAfterViewInit` hook.

**Comment by ottosson** [comment]
Strange... So what's up with the StackBlitz example then?

**Comment by AMBudnik** [comment]
It might be related to StackBlitz itself (having some temporary issues).

This is what I get on Safari and Chrome

![Zrzut ekranu 2020-10-12 o 11 15 35](https://user-images.githubusercontent.com/10757813/95728907-61ba2800-0c7c-11eb-98d5-f9961618f20c.png)

**Comment by ottosson** [comment]
Works for me in Chrome... and the behavior occurred at first in my Electron application, which I could reproduce in SB.

**Comment by AMBudnik** [comment]
I was able to get it working on Firefox (it seems to be a cache error on my side).

This is the result of the demo I got from you

![Zrzut ekranu 2020-10-12 o 11 23 44](https://user-images.githubusercontent.com/10757813/95729653-76e38680-0c7d-11eb-8007-52ecb71afa18.png)

We can see that the `getSettings()` returns the data and the settings of the prototype. The prototype holds the default and changed values for options.  This is the same as what see in the JSFiddle demo. The prototype is overwritten. We can see it (for example) by checking the `colHeaders` that by default are `null` 

![Zrzut ekranu 2020-10-12 o 11 34 25](https://user-images.githubusercontent.com/10757813/95730706-e9089b00-0c7e-11eb-892c-31288bc0ef6e.png)
_(part of the `getSettings` call for Handsontable built with no options)_

but changes to `true` as column headers have been defined in the .HTML file.

![Zrzut ekranu 2020-10-12 o 11 33 45](https://user-images.githubusercontent.com/10757813/95730620-cd9d9000-0c7e-11eb-827b-68181157ec22.png)
